src/gsi/read_satwnd.f90

```diff
@@ -1209,6 +1217,11 @@ subroutine read_satwnd(nread,ndata,nodata,infile,obstype,lunout,gstime,twind,sis
                     c_station_id='WV'//stationid
                     c_sprvstg='WV'
                     !write(6,*)'itype= ',itype
+                 else if(trim(subset) == 'NC005099')  then            ! WV clear sky/deep layer
+                    itype=241
+                    c_station_id='IR'//stationid
+                    c_sprvstg='IR'
+                    !write(6,*)'itype= ',itype
```

Comment:
Is it necessary to keep line 1224 here?

Reply:
Yes, I will delete  !write(6,*) 'NC005099 readin' .
However, for line 1224, if you mean the "!write(6,*)'itype= ',itype", it
seems like other "elseif" block above (e.g. line 1219, line 1214, etc)
keeps this one, so to be consistent, we have this line here. But if you
think it is not necessary, I will delete the line as well.